app/views/notices/shop_employer_notices/initial_employer_reminder_to_publish_plan_year.html.erb

```diff
@@ -0,0 +1,122 @@
+<%= render partial: 'notices/shared/logo' %>
+<%= render partial: 'notices/shared/date', :locals => {notice: notice} %>
+<%= render partial: "notices/shared/address", :locals => {notice: notice} %>
+
+<% case notice.notification_type %>
+<% when "initial_employer_first_reminder_to_publish_plan_year" %>
+  <% subject = "Action Required to Complete #{notice.primary_fullname}’s Application for Employer-sponsored Coverage with #{Settings.site.short_name}" %>
+<% when "initial_employer_second_reminder_to_publish_plan_year" %>
+  <% subject = "Action Required to Complete #{notice.primary_fullname}’s Application for Employer-sponsored Coverage with #{Settings.site.short_name} – Deadline Extended" %>
+<% when "initial_employer_final_reminder_to_publish_plan_year" %>
+  <% subject = "Action Required to Complete #{notice.primary_fullname}’s Application for Employer-sponsored Coverage with #{Settings.site.short_name}" %>
+<% end %>
+
+<table class="total_table" border="0">
+  <tbody>
+    <tr>
+      <td><h3> <%= subject %> </h3></td>
+    </tr>
+  </tbody>
+</table>
+
+<table class="total_table" border="0">
+  <tbody>
+    <tr>
+      <td>Dear <%= notice.primary_fullname %>:</td>
+    </tr>
+  </tbody>
+</table>
+
+<% case notice.notification_type %>
```

Comment:
I do not get it All these 3 case statements are doing same. can we do something like this when "initial_employer_first_reminder_to_publish_plan_year" || "initial_employer_second_reminder_to_publish_plan_year" ||
"initial_employer_final_reminder_to_publish_plan_year"